packages/MarkdownEditor-Core.package/MarkdownInlineTextStyler.class/instance/initialize.st

```diff
@@ -4,5 +4,4 @@ initialize
 	super initialize.
 	self 
 		clean;
-		tokenClasses: {MarkdownEmphasisDelimiter. MarkdownString} asSortedCollection.
-	self tokenClasses sortBlock: [:a :b | a precedence > b precedence]
\ No newline at end of file
+		tokenClasses: {MarkdownEmphasisDelimiter. MarkdownString}
```

Comment:
There is really no reason to have this as an instance variable. Either the accessor could be hardcoded or since the accessor is only used once you could have the collection in the code there. 
(We also did this in the comment block PR since we wanted to make it consistent to the Block selection in the BlockTextStyler so I'm looking forward to merge conflicts :D)

Reply:
We could even do (which encodes the catch-all behaviour better)
```smalltalk
tokenClass := (MarkdownEmphasisDelimiter mayBeginWith: aCharacter)
	ifTrue: [MarkdownEmphasisDelimiter]
	ifFalse: [MarkdownString].
```

That way, `mayBeginWith:` does not need to be implemented in `MarkdownString` and can be removed as `subclassResponsibility`

Reply:
Yea I guess since we only have two kinds of tokens this makes sense. My thinking was to make it as consistent with the BlockStyler as possible, but simple design dictates your solution.

Reply:
If consistency is important to you, I'm also fine with
```smalltalk
matchingTokenClass := {MarkdownEmphasisDelimiter}
	detect: [:tokenClass | tokenClass mayBeginWith: aCharacter]
	ifNone: [MarkdownString].
```
It's a bit more code and has a single-element collection, but it is the same pattern like in block parsing. Which would you take now?

Reply:
I like the consistency better.